src/Pages/Explore/Explore.js

```diff
@@ -1,7 +1,69 @@
+import { useCategory } from "../../Context/CategoryContext"
+import { useFilter } from "../../Context/FilterContext"
+import { useVideo } from "../../Context/VideoContext"
+import {Filter} from '../../Reducer/Filter'
+import './Explore.css'
+import { useState } from "react"
 export const Explore = () =>{
+    const [isActive, setIsActive] = useState('All')
+    const { categories} = useCategory()
+    const { videos} = useVideo()
+    const { state, dispatch} = useFilter()
+   
+    const handleButton = () =>{
+        setIsActive('All')
+        dispatch({
+            type:"ALL"
+        })
+    }
+   const sorted = Filter(videos, state)
+  
     return(
-        <div>
-            explore page
+        <div className="m-2">
```

Comment:
please gave some meaningful class name

Reply:
This is utility class that i am using from component library